kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -79,7 +79,8 @@ def from_json(cls, json_string: str):
         items = list_from_json(cls.__name__, json_string)
         obj = cls()
         for i in items:
-            obj.append(from_dict(data_class=_Alarm, data=i))
+            if "type" not in i:
```

Comment:
Please add a comment that it is a workaround for "problematic" payloads

Reply:
Will do.

Reply:
I've added comment and at the same time made the workaround better. The `alerting.get_active_alerts` call can now return also manual mitigations :). This code needs deeper refactoring.